README.md

```diff
@@ -71,115 +67,48 @@ The widget generator will include a **Gulpfile.js** and the necessary package.js
 
 Earlier versions of the widget generator added Grunt as the default taskrunner. We included this option as well if you want to use this.
 
-\* **We do not recommend using jQuery in your widgets, please [read this issue](https://github.com/mendix/AppStoreWidgetBoilerplate/issues/38). For simple DOM-manipulation you can use Dojo, which is provided by Mendix. Only use jQuery when you need it for certain external libraries that depend on it.**
-
 ---
 
-## Use a taskrunner for development
-
-The widget generator will scaffold a widget for you. It provides a convenient setup to develop you widget and publish it on Github. To make it even more easy, we add files for taskrunners.
+### 2.2. Which template do you want to use for the widget?
 
+#### BadgeWidgetBoilerplate
 
-### Gulp
+The badge boiler plate is a react fully developed mendix widget that Shows a value as a badge or a color label which is recomended for beginners. Its has the following features
```

Comment:
- Correct `Shows` to `shows`
- Correct  `recomended ` to `recommended`
- Consider the re-write: `The badge boiler plate is a react fully developed mendix widget that shows a value as a badge or a color label and is recommended for beginners. It has the following features:-`

Reply:
Done